src/rankcorr.jl

```diff
@@ -10,9 +10,19 @@
 #
 #######################################
 
+"""
+    corspearman(x, [y])
```

Comment:
Actually I realize it might be more explicit to write the default as `y=x`. Same for other correlation functions.